test/setupVitest.ts

```diff
@@ -13,3 +13,21 @@ failOnConsole({
 });
 
 afterEach(cleanup);
+
+// Mocks
+
+if (typeof window !== 'undefined') {
```

Comment:
You also need to check whether `matchMedia` exists. We run these tests in real browsers as well